### PR TITLE
chore: readme improvements

### PR DESCRIPTION
**Pull request by matthewelwell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+from flagsmith import Flagsmith
+from openfeature_flagsmith.provider import FlagsmithProvider
+
+provider = FlagsmithProvider(
+    # Provide an instance of the Flagsmith python client.
+    # Required: True
+    client=Flagsmith(...),
+    
+    # By enabling the use_flagsmith_defaults setting, you can instruct the OpenFeature SDK to use
+    # the default logic included in the Flagsmith client. This will override the default provided
```
Comment (dabeeeenster) [inline README.md:31]: maybe a link here to what the 'default logic' is - unclear to me atm

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+    # at evaluation time in the OpenFeature SDK in most cases (excluding those where an unexpected
+    # exception happens in the Flagsmith client itself).
+    # Required: False
+    # Default: False
+    use_flagsmith_defaults=False,
+    
+    # By default, when evaluating the boolean value of a feature, the Flagsmith OpenFeature Provider
+    # will use the 'Enabled' state of the flag. This behaviour can be changed to use the value
```
Comment (dabeeeenster) [inline README.md:39]: What is "value" here?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,57 @@
...
+    # defined in the Flagsmith feature instead by enabling the use_boolean_config_value setting.
+    # Required: False
+    # Default: False
+    use_boolean_config_value=False,
```
Comment (zachaysan) [inline README.md:43]: The wording here is hard to understand clearly. 

> when evaluating the boolean value of a feature, the Flagsmith OpenFeature Provider
> will use the 'Enabled' state of the flag.

Then it didn't really evaluate the boolean value of a feature, right? It just looks at the enabled state.

**Review by zachaysan — APPROVED** [review]
One section is hard to follow, but overall good 👍🏼

File: README.md (hunk 2 above)
Comment (matthewelwell) [reply README.md:39]: Not sure how to make this more clear tbh, it's referring to the 'value' field in the Flagsmith feature: 

image

Any suggestions?

File: README.md (hunk 1 above)
Comment (matthewelwell) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (matthewelwell) [reply README.md]: I've tried my best to reword it so it's clearer.

File: README.md (hunk 3 above)
Comment (matthewelwell) [reply README.md]: I've tried my best to reword it so it's clearer.

**Review by dabeeeenster — APPROVED** [review]
